pydnd/equipment/views.py

```diff
@@ -35,6 +35,31 @@ def get(self, request, name_or_id):
         return Response(model_to_dict(queryset), status=status.HTTP_200_OK)
 
 
+class EquipmentSubCategoryGet(APIView):
+
+    def get(self, request, name_or_id):
```

Comment:
Lets think of error cases here. Maybe put some of this in a try/except and return a 404. Maybe make the object getters be the object or 404 method